FPR-0089/EVAL-0089.md

```diff
@@ -0,0 +1,57 @@
+
+# FPR Evaluation of FPR-0089
+
+Status - Partially Accepted - Changes required
+
+- [ ] **1. Has the FPR been submitted correctly?**
+   - [ ] A. Correctly titled?
+   - [ ] B. Used the full FPR template?
+   - [ ] C. Pull request correctly made to the BCF FPR repo?
+
+- [ ] **2. Have all sections of the template been completed?**
+   - [ ] A. Project Name
+   - [ ] B. Start Date
+   - [ ] C. Hashtag
+   - [ ] D. Name of BCF Chat room
+   - [ ] E. Stakeholders
+   - [ ] F. Project Summary
+   - [ ] G. Resources
+   - [ ] H. Budget
+   - [ ] I. Timeline
+   - [ ] J. Goals
+   - [ ] K. Other
+
+- [ ] **3. Is it well formatted for legibility?**
+
+- [ ] **4. Does the project have a non-profit purpose?**
+
+- [ ] **5. Has it been completed with enough detail for the scale of the project?**
+
+- [ ] **6. Is the team trustworthy enough for the funding requirement?**
+
+- [ ] **7. Are the objectives S.M.A.R.T?**
+   - [ ] A. Specific
+   - [ ] B. Measurable
+   - [ ] C. Agreed Upon
+   - [ ] D. Realistic
+   - [ ] E. Time-bound
+
+- [ ] **8. Are the stakeholders realistically providing volunteered time?**
+
+- [ ] **9. Is the cost vs impact satisfactory?**
+
+Cost = TODO
+
+Impact
```

Comment:
The impact would be first at the national level, as I have already explained, of going to the 24 cities, and also international, because we would put Venezuela as a country that accepts bitcoin cash.